app/src/main/java/com/zeltixdev/photoapp/ui/photo/list/adapter/viewholder/PhotoViewHolder.kt

```diff
@@ -0,0 +1,42 @@
+package com.zeltixdev.photoapp.ui.photo.list.adapter.viewholder
+
+import androidx.recyclerview.widget.RecyclerView
+import androidx.swiperefreshlayout.widget.CircularProgressDrawable
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.zeltixdev.photoapp.databinding.PhotoListItemBinding
+import com.zeltixdev.photoapp.entity.Photo
+
+private const val STROKE = 5f
+private const val CENTER_RADIUS = 30f
+
+class PhotoViewHolder(
+    private val binding: PhotoListItemBinding,
+    onPhotoIdClick: (String) -> Unit
```

Comment:
I noticed you only need the photo id on Fragment side, that's why I'm sending it only that and no more.